utils/notionTypeParser/manualsPreviewParser.js

```diff
@@ -0,0 +1,31 @@
+import tp from '../typograf/typograf.config'
+
+import HIDDEN_MANUALS from '../../consts/hidden-manuals'
+
+function parseManualsPreview(tree) {
+    // TODO Remove hard-code after back-end feature https://github.com/ekaterinburgdev/guides-api/issues/10
+    const manualsVisible = tree.children.filter((manual) => {
+        return !HIDDEN_MANUALS.includes(manual?.properties?.pageUrl?.url)
+    })
+
+    return manualsVisible
+        .sort((a, b) => {
+            const orderA = a.properties?.order?.number
+            const orderB = b.properties?.order?.number
+            return orderB - orderA
+        })
+        .map(({ properties }) => {
+            return {
+                title: tp.execute(properties?.Name?.title[0]?.text?.content || ''),
+                subtitle: tp.execute(properties?.subtitle?.rich_text[0]?.plain_text || ''),
```

Comment:
It may be possible to do a destructuring ?

const { Name, color, status, publishedDate } = properties

Reply:
done